pages/_type/_id.vue

```diff
@@ -812,15 +832,29 @@
           <div class="infos">
             <div class="info">
               <div class="key">License</div>
-              <div class="value uppercase">
+              <div class="value lowercase">
                 <a
                   v-if="project.license.url"
                   class="text-link"
                   :href="project.license.url"
                 >
+                  {{ project.license.id.replaceAll('LicenseRef-', '') }}
+                </a>
+                <a
+                  v-else-if="
+                    project.license.id === 'LicenseRef-All-Rights-Reserved'
+                  "
+                  class="text-link"
+                  @click="getLicenseData()"
+                >
+                  All Rights Reserved
```

Comment:
see other comments about this section in the other side of the page, also "All Rights Reserved" => ARR